File: app/dbmodels/NetworkPortEthernet.php
<?php

$table = array(
    'fields'        => array(),
    'oldfields'     => array(),
    'renamefields'  => array(),
    'indexes'       => array(),
    'oldindexes'    => array(),
    'relationships' => array(),
    'model'         => 'NetworkPortEthernet',
    'menu'          => ''
);
$table['fields'] = array(
    'id'                     => DBModels::type('increments',
                                               array('visible' => false)),
    'network_port_id'        => DBModels::type('integer'),
    'device_network_card_id' => DBModels::type('integer'),
    'netpoint_id'            => DBModels::type('integer'), 
    'type'                   => DBModels::type('string'), /// COMMENT 'T, LX, SX',
    'speed'                  => DBModels::type('integer',array('default'=>10)), /// COMMENT 'Mbit/s: 10, 100, 1000, 10000',

);


$table['relationships'] = array(
    'network_port' => array(
        'type'  => 'belongsTo',
        'item'  => 'NetworkPort'),
    'device_network_card' => array(
        'type'  => 'belongsTo',
        'item'  => 'DeviceNetworkCard'),
    'netpoint' => array(
        'type'  => 'belongsTo',
        'item'  => 'Netpoint'),
);

File: fusionglpi/database/networkport_ethernet.py
from fusionglpi import db

class NetworkportEthernet(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    networkport_id = db.Column(db.Integer, db.ForeignKey('networkport.id'))
    device_networkcard_id = db.Column(db.Integer, db.ForeignKey('device_networkcard.id'))
    netpoint_id = db.Column(db.Integer, db.ForeignKey('netpoint.id'))
    type = db.Column(db.String(255, None, True))
    speed = db.Column(db.Integer, nullable=False, server_default="10")
